code/modules/modular_computers/file_system/programs/radar.dm

```diff
@@ -0,0 +1,235 @@
+/datum/computer_file/program/radar //generic parent that handles most of the process
+	filename = "genericfinder"
+	filedesc = "debug_finder"
+	ui_header = "borg_mon.gif" //DEBUG -- new icon before PR
+	program_icon_state = "generic"
+	extended_desc = "generic"
+	requires_ntnet = TRUE
+	transfer_access = null
+	available_on_ntnet = FALSE
+	network_destination = "tracking program"
+	size = 5
+	tgui_id = "NtosRadar"
+	ui_x = 800
+	ui_y = 600
+	special_assets = list(
+		/datum/asset/simple/radar_assets,
+	)
+	var/list/objects //list of things that are trackable
+	var/atom/selected //trackable thing selected by the user
+	var/scanning = FALSE //To disable scan button while scan is running
```

Comment:
[Please add DMDOC which matches the required standard](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)